Add additional documentation for code highlighting

matalo33 opened the pull request:
Fixes #283

kyeotic commented:
@matalo33 I dont understand how this fixes #283. That issue indicates that the problem is with the highlight pack itself not containing common languages, but the documentation you added doubles down on the confusion that it _does_ contain the common languages. Has the highlight pack been updated by another PR? If not, the documentation solution is to explain to users how to generate and override the highlight pack with one that contains languages like JS

matalo33 commented:
The highlight.pack.js was updated today in commit https://github.com/matcornic/hugo-theme-learn/commit/8349d619a52521c4a314191a0384824a58ca717a. Admittedly it should have been in this PR but I included it in the wrong one by accident.

As I understand #283 it's stating there are missing common languages (now fixed) and that the docs to support additional languages was missing (also now fixed).

Did I miss anything else?

kyeotic commented:
Nope, sounds good. The highlight pack update was the missing piece
